README.md

```diff
@@ -1,117 +1,69 @@
-Node Test and Build Infrastructure
-==================================
+Node.js Build Working Group
+===========================
 
-Chat with us! We use IRC: [#node-build at freenode](irc://irc.freenode.net/node-build).
+Chat with us! We use IRC: [#node-build at Freenode](irc://irc.freenode.net/node-build)
 
 Wut?
 ----
 
```

Comment:
How about adding a sentience about the build group?

`The node.js build working group maintains and controls infrastructure used for continuous integration, releases, benchmarks and more. Our mission is to provide node.js with a solid testing infrastructure that aims to improve the quality of the software itself by targeting correctness, speed or cross-compatibility issues.`